SF iOS/SF iOS/Feed/SwiftUI/Data.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  Data.swift
+//  Coffup
+//
+//  Created by Roderic Campbell on 6/8/19.
+//
+
+import UIKit
+import SwiftUI
+import CoreLocation
+import Combine
+
+struct WrappedEvent /*, BindableObject*/ {
+//    var didChange = (NotificationCenter.default.publisher(for: "eventChange") as AnyObject).receive(on: RunLoop.main)
+//    var event: Event
+    let name: String
+    let eventID: String
+    let startDate: String
+    let venue: Venue
+    var locationCoordinate: CLLocationCoordinate2D {
+        CLLocationCoordinate2D(
+            latitude: venue.location.latitude,
+            longitude: venue.location.longitude)
+    }
+    
+    enum CodingKeys: String, CodingKey {
+        case eventID = "id"
+        case name = "name"
+        case venue = "venue"
+        case startDate = "start_at"
+    }
+}
```

Comment:
^ That's a new Event object. The idea was to "wrap" the Objc event, but it was pretty easy to just write it